packages/react/src/internal/useAnnouncer.js

```diff
@@ -5,9 +5,12 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-export function useAnnouncer(textCount, maxCount) {
+export function useAnnouncer(textCount, maxCount, entityName) {
```

Comment:
if we set the default of `entityName `to 'characters' here then the if statement below is not necessary and we can always construct the sentence using `${entityName}`
```suggestion
export function useAnnouncer(textCount, maxCount, entityName = 'characters') {
```

Reply:
done